### DISCUSSION TITLE
What is a unit of measurement for the smallest mouse cursor movement? 🐭

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1263] [Galaxy Trivia] [answered]
**Bonus:** Crimson is a shade of which color? *(share hex)*

**Comment by tomerf** [comment, marked as answer]
A mickey is a unit used to describe the smallest detectable distance of the movement of a computer mouse (presumably getting its name from Mickey Mouse). It is approximately 1/200 of an inch.

Bonus: Crimson is a shade of deep purplish-red with the hex code #DC143C.
